packages/expo-document-picker/src/types.ts

```diff
@@ -21,6 +21,14 @@ export type DocumentPickerOptions = {
    *
    */
   multiple?: boolean;
+  /**
+   * If set to `true`, the system UI will use the `ACTION_GET_CONTENT` intent instead of
+   * `ACTION_OPEN_DOCUMENT` to allow the user to pick a file. This is useful when you want to pick
+   * a file from other sources than the document providers, like Google Drive or Dropbox.
```

Comment:
```suggestion
  /**
   * If set to `true`, the system UI uses the `ACTION_GET_CONTENT` intent instead of
   * `ACTION_OPEN_DOCUMENT` to allow the user to pick a file. This is useful when you want to pick
   * a file from sources other than the document providers, such as Google Drive or Dropbox.
```

A couple of suggestions as per our writing style guide. One key highlight: [Using present tense where possible](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#use-present-tense).